cargo-crev/src/repo.rs

```diff
@@ -192,6 +188,12 @@ fn build_graph<'a>(
                 .dependencies()
                 .iter()
                 .filter(|d| d.matches_ignoring_source(raw_dep_id))
+                .filter(|d| {
+                    let is_local = !d.source_id().is_registry();
+                    // Dev/build dependencies can lead to circular dependencies (in combination with normal deps),
+                    // so ignore dev deps on local crates, as it's not helpful anyway
+                    d.kind() == DepKind::Normal || (dev_dependencies && !is_local)
```

Comment:
should it allow `DepKind::Build` here?

Reply:
I don't think so. Both test and build might introduce circular dependency, so should be ignored between the local crates.